orc8r/tools/packer/magma-test-virtualbox.json

```diff
@@ -89,11 +89,6 @@
       "script": "scripts/vagrant_key.sh",
       "type": "shell"
     },
-    {
-      "execute_command": "echo 'vagrant' | {{.Vars}} sudo -S -E bash '{{.Path}}'",
-      "script": "scripts/setup.sh",
```

Comment:
This was removed with the comment that `ubuntu_setup.sh` already covers all the steps - I do not see `ubuntu_setup.sh` called here directly. Is it called implicitly?

Reply:
It's called in L71.

Reply:
woops, (what did I search for?), merci